client/components/popover/menu-item.jsx

```diff
@@ -14,14 +28,21 @@ var MenuItem = React.createClass( {
 	},
 
 	render: function() {
-		var onMouseOver = this.props.focusOnHover ? this._onMouseOver : null;
+		const onMouseOver = this.props.focusOnHover ? this._onMouseOver : null;
+
+		let icon;
+		if ( this.props.icon ) {
```

Comment:
>Why not a single line below instead of `{ icon }`?

Equally valid, and on a different day I might have done the latter 😄  I'm fine to change it.